modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -531,30 +537,36 @@ def validate_claim_date_to_active_duty_end_date!(service_information)
         end
         max_active_duty_end_date = max_period['activeDutyEndDate']
 
```

Comment:
We get a 500 if the `activeDutyEndDate` is not present here

Reply:
@stiehlrod I believe [API-32142](https://jira.devops.va.gov/browse/API-32142) will be dealing with this specific case since the `activeDutyBeginDate` and `activeDutyEndDate` will become required once that ticket goes in.